Changes.md

```diff
@@ -2,6 +2,16 @@
 
 [Sidekiq Changes](https://github.com/sidekiq/sidekiq/blob/main/Changes.md) | [Sidekiq Pro Changes](https://github.com/sidekiq/sidekiq/blob/main/Pro-Changes.md) | [Sidekiq Enterprise Changes](https://github.com/sidekiq/sidekiq/blob/main/Ent-Changes.md)
 
+HEAD (7.3.0)
+----------
+
+- Allow `Sidekiq::Limiter.redis` to use Redis Cluster [#6288]
+- Adjust CSP to disallow inline styling and scripts within the Web UI [#6270]
```

Comment:
Inline styling got reverted and inline scripts are already disallowed (#6074). This will be more accurate:

> Adjust CSP to disallow remote scripts within the Web UI.
> ... 3rd party extensions that load scripts from something like a CDN will need to adjust their web assets accordingly.

Also, perhaps something like:

```erb
# Before
<script type="text/javascript" src="https://code.jquery.com/jquery-3.7.1.min.js"></script>
# After
<script type="text/javascript" src="https://code.jquery.com/jquery-3.7.1.min.js" <%= defined?(csp_nonce) ? "nonce=\"#{csp_nonce}\"" : "" %>></script>
```

This will work in both 7.2 and 7.3 so extensions wouldn't have to worry about that.